lib/shrinkwrapper.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+var gutil = require('gulp-util');
+var fs = require('fs');
+var path = require('path');
+var spawn = require('child_process').spawnSync;
+var isWindows = /^win/.test(process.platform);
+var npmCmd = isWindows
```

Comment:
Shouldn't be necessary; Windows users don't have to type `npm.cmd`; `.cmd`, like `.exe`, is automatically searched for when executing a binary.